Controller/CRUDController.php

```diff
@@ -157,7 +157,7 @@ public function batchActionDelete(ProxyQueryInterface $query)
     public function deleteAction($id)
```

Comment:
I wanted to suggest actually deleting the parameter in the action function as it is not used and a bit misleading

Reply:
I'll open a separate issue for this so it is at least logged

Reply:
See #3970